packages/template-retail-react-app/app/components/quantity-picker/index.jsx

```diff
@@ -41,12 +41,24 @@ const QuantityPicker = (props) => {
 
             // If there is a `onFocus` property define, call it with the event captured.
             // eslint-disable-next-line react/prop-types
-            onFocus && onFocus.call(this, e)
+            onFocus?.call(this, e)
         }
     })
 
-    const inc = getIncrementButtonProps({variant: 'outline'})
-    const dec = getDecrementButtonProps({variant: 'outline'})
+    const inc = getIncrementButtonProps({
+        variant: 'outline',
+        'aria-label': intl.formatMessage({
+            defaultMessage: 'Increment Quantity',
+            id: 'product_view.label.aria.quantity_increment'
```

Comment:
Aside: I think we're not being consistent with the message ids. Sometimes we say "aria", and previously we used "assistive_msg".

Feel free to leave things as they are for now.